### DISCUSSION TITLE
Reviewing others work

### DISCUSSION THREAD

**Discussion by BioNCphilic** [earthlab-education/Earth-Analytics-AY24#352] [General]
I'm just curious about when people are assigned to review work done by others. I know timing is complicated by the fact that we are not all in the same section, but I'd like to close out the First Map #174 assignment. I'm not assigned to review anyone and I'm traveling for the next 10 days and a bit nervous about getting behind. Any chance I can assign myself to review others and be done with it?

**Comment by eculler** [comment]
Hi @BioNCphilic -- first off, if you don't want to worry about reviews while you're away, that's fine. They're incorporated into your participation score as one of several ways to earn participation (adding/answering discussion posts, attending discussion section or office hours, and responding to the reading when we have it are other ways). I'm looking for an average of two types of participation each week (10 points), and you can earn up to 20 or 25 points each week depending on whether there's assigned reading.

That said, the reviews are assigned in a round robin. If you haven't come up yet,  because not everyone has submitted yet, and you want to review something, please feel free to add a review of a post you are not assigned to! As long as you leave a review (rather than a comment), I should be able to find it. NOTE that if you are not assigned, you will not see a button to leave a review on the main page of the Pull Request, but if you go to Files Changed you will still have the option to leave a review there!
